hw/usb/tinyusb/std_descriptors/include/tusb_config.h

```diff
@@ -118,7 +153,13 @@ extern "C" {
 #define CFG_CDC_CONSOLE          0
 #endif
 
-#define CFG_TUD_CDC              ((CFG_CDC) + (CFG_CDC_CONSOLE))
+#if MYNEWT_VAL(USBD_CDC_HCI)
```

Comment:
Is USBD_CDC_I defined in any syscfg.yml file? I could not find it but maybe it is in a different PR or commit or I missed it?